package.json

```diff
@@ -1,98 +1,28 @@
 {
-  "name": "forklift-console-plugin",
-  "version": "0.0.1",
+  "name": "@monorepo/root",
   "private": true,
+  "version": "0.0.1",
   "repository": "git@github.com:kubev2v/forklift-console-plugin.git",
   "license": "Apache-2.0",
+  "workspaces": [
+    "./packages/*"
+  ],
   "scripts": {
-    "clean": "rm -rf dist",
-    "build": "yarn clean && NODE_ENV=production yarn webpack",
-    "start": "yarn webpack serve",
+    "start": "npm run start -w @kubev2v/forklift-console-plugin",
+    "build": "NODE_ENV=production npm run build -ws",
+    "clean": "rm -rf dist && rm -rf coverage && npm run clean -ws",
+    "clean:all": "npm run clean && rm -rf node_modules && npm run clean:all -ws",
     "start:console": "source ./scripts/configure/openshift.sh && ./scripts/start-console.sh",
     "start:console:oauth": "source ./scripts/configure/openshift-auth.sh && ./scripts/start-console.sh",
-    "i18n": "i18next \"src/**/*.{js,jsx,ts,tsx}\" [-oc] -c i18next-parser.config.mjs",
-    "lint": "eslint . && stylelint \"src/**/*.css\" --allow-empty-input",
-    "lint:fix": "eslint . --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
-    "test": "TZ=UTC jest",
+    "i18n": "i18next \"packages/**/*.{js,jsx,ts,tsx}\" [-oc] -c packages/forklift-console-plugin/i18next-parser.config.mjs",
+    "lint": "eslint . && stylelint \"packages/**/*.css\" --allow-empty-input",
+    "lint:fix": "eslint . --fix && stylelint \"packages/**/*.css\" --allow-empty-input --fix",
+    "test": "npm run test -w @kubev2v/forklift-console-plugin",
     "test:i18n": "./scripts/test-i18n.sh",
-    "test:coverage": "jest --coverage",
-    "test:updateSnapshot": "jest --updateSnapshot",
+    "test:coverage": "TZ=UTC jest --coverage",
+    "test:updateSnapshot": "TZ=UTC jest --updateSnapshot",
```

Comment:
Apart from the `start:console` scripts, the others look like they need to be better adapted to the various packages.

Does it make sense to run the linter from the monorepo root across all packages, or would you want the linters in each package to be able to run independently?  I'm leaning toward the latter.

Reply:
moved to running linting on each package
will move the tests to also run on each package individually